js/main.js

```diff
@@ -0,0 +1,227 @@
+// const $graphButton = document.querySelectorAll('[button = "view-graph"]');
+const $homeDiv = document.querySelector('[data-view = "home-screen"]');
+const $gratefulDiv = document.querySelector('[data-view = "grateful"]');
+const $5thingsForm = document.getElementById('five-things');
+const $journalTextForm = document.querySelector('#journal-cont');
+const $headerUl = document.querySelector('.header-list');
+const $headerLogo = document.querySelector('#header-logo');
+const $dateH2 = document.querySelectorAll('[data="date"]');
+const $saveDraftButton = document.querySelectorAll('[button = "save-draft"]');
+const $nJContButton = document.querySelector('[button = "new-journal-cont"]');
+const $grateful1 = document.getElementById('grateful1');
+const $grateful2 = document.getElementById('grateful2');
+const $grateful3 = document.getElementById('grateful3');
+const $grateful4 = document.getElementById('grateful4');
+const $grateful5 = document.getElementById('grateful5');
+const $fiveThings = [$grateful1, $grateful2, $grateful3, $grateful4, $grateful5];
+const $NJDiv = document.querySelector('[data-view="new-journal"]');
+const $NJTextCont = document.querySelector('#journal-cont-text');
+const $doneButton = document.querySelector('[button="done"]');
+const $modalDiv = document.querySelector('[data-view="done-modal"]');
+const $scoreH1 = document.querySelector('[data="modal-score"]');
+const $modalQuoteP = document.querySelector('[data="modal-quote"]');
+const $homeButton = document.querySelector('[button="home"]');
+const $pgList = [$gratefulDiv, $NJDiv, $modalDiv];
+let date;
+let formattedDate;
+
+function showPage(show, hide) {
+  show.setAttribute('class', 'container');
+  hide.setAttribute('class', 'hidden');
+}
+
+function getDate() {
+  const monthArr = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
+  date = new Date();
+  let month = date.getMonth();
+  month = monthArr[month];
+  let day = date.getDate();
+  day = day.toString();
+  let year = date.getFullYear();
+  year = year.toString();
+  formattedDate = month + ' ' + day + ', ' + year;
+}
+
+function removePageID() {
+  if (($headerUl.childNodes) > 3) {
+    const $pgID = document.querySelector('[data = "pg-ID"]');
+    $headerUl.removeChild($pgID);
+  }
+}
+
+function getScoreNum(score) {
+  if (score === 'P+') {
+    // eslint-disable-next-line no-undef
+    currentObj.scoreNum = 2;
+  } else if (score === 'P') {
+    // eslint-disable-next-line no-undef
+    currentObj.scoreNum = 1;
+  } else if (score === 'NEU') {
+    // eslint-disable-next-line no-undef
+    currentObj.scoreNum = 0;
+  } else if (score === 'N') {
+    // eslint-disable-next-line no-undef
+    currentObj.scoreNum = -1;
+  } else if (score === 'N+') {
+    // eslint-disable-next-line no-undef
+    currentObj.scoreNum = -2;
+  } else {
+    // eslint-disable-next-line no-undef
+    currentObj.scoreNum = null;
+  }
+}
```

Comment:
Likewise, instead of disabling the `no-undef` rule over and over again, add a single `/* global ... */` linter comment at the top specifying which variables come from another file: https://eslint.org/docs/user-guide/configuring/language-options#specifying-globals